res/css/structures/_UserMenu.scss

```diff
@@ -208,15 +208,49 @@ limitations under the License.
 .mx_UserMenu_CustomStatusSection {
     margin: 0 12px 8px;
 
-    .mx_UserMenu_CustomStatusSection_input {
+    .mx_UserMenu_CustomStatusSection_field {
         position: relative;
         display: flex;
 
-        > input {
+        &.mx_UserMenu_CustomStatusSection_field_focused,
```

Comment:
is this really needed vs `:focus`?

Reply:
In this case no. Can update ⏩

For [search](https://github.com/matrix-org/matrix-react-sdk/blob/d00809b8b9023230c4c34b30f063288e1fe44608/res/css/structures/_RoomSearch.scss#L40-L92), it was used because the `<input>` is wrapped in a fake looking input to encapsulate everything and `display: flex;` it.

Reply:
Thinking about it more. Would you prefer a sibling selector over this?

```scss
> .mx_UserMenu_CustomStatusSection_input:focused + .mx_UserMenu_CustomStatusSection_clear {
    display: block;
}
```